### PR TITLE
fix: [M3-6188] - Fix TopMenuIcon tooltips

### PR DESCRIPTION
**Pull request by carrillo-erik**
## Description 📝
Fixes `TopMenu` issues via the `tab` key navigation which resulted in the `ToolTip` not being shown in the following icons: `TopMenuIcon`, `Help`, and `Community`.

## Major Changes 🔄
- Corrects the function name from `Help` to `Community` in the `Community.tsx` file.
- Replaces default exports with named exports.
- Removes usage of the `Link` component and replaces it with the `StyledLinkButton` component.
- Adds verbiage to the `Community` icon tooltip to inform the user that the link will open in a new tab.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Img-00 | Img-02 |
| Img-01 | Img-03 |

## How to test 🧪
1. Navigate the `TopMenu` via the `tab` key and inspect that the appropriate tooltip is shown upon focus.
2. Hover over the`TopMenu` icons and inspect that the appropriate tooltips are shown upon accordingly.
3. Click or press the `enter`/`return` key on the `Help` and `Community` icons to test that each link works as intended

### CODE REVIEW

File: packages/manager/src/features/TopMenu/Help.tsx [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+        aria-label="Help & Support"
+        role="link"
+        onClick={() => history.push('/support')}
       >
-        <TopMenuIcon title={'Help & Support'}>
-          <TooltipIcon status="help" />
-        </TopMenuIcon>
-      </StyledTopMenuIconWrapper>
-    </Link>
+        <StyledTopMenuIconWrapper sx={{ marginLeft: [null, null, '8px'] }}>
```
Comment (jaalah-akamai) [inline packages/manager/src/features/TopMenu/Help.tsx:17]: What is `marginLeft: [null, null, '8px']`?

**Review by dwiley-akamai — APPROVED** [review]
functionality LGTM

**Review by jaalah-akamai — APPROVED** [review]

**Review by abailly-akamai — APPROVED** [review]
Nice! Approved pending cleanup comment

File: packages/manager/.changeset/pr-9371-fixed-1688749077631.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Fixed
+---
+
+Fixed TopMenu bug that prevented tooltip from showing via tab key navigation. ([#9371](https://github.com/linode/manager/pull/9371))
```
[the remark this answers is not in the document]
Comment (carrillo-erik) [reply packages/manager/.changeset/pr-9371-fixed-1688749077631.md:5]: Thank you, I hadn't noticed this.

**Pull request merged**